spec/support/page_objects/teacher_interface/application/form_page_objects/age_range_form.rb

```diff
@@ -0,0 +1,8 @@
+module PageObjects
```

Comment:
A lot of these form objects are similar, if not identical, with a heading and some grid rows. I'd suggest the common parts could be refactored out in to a `PageObject::TeacherInterface::ApplicationForm` object and the ones which require specific elements could inherit from it.